carbon/assetlist.json

```diff
@@ -160,22 +118,85 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/bneo.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/bneo.svg"
       },
-      "images": [
+      "coingecko_id": "neo"
+    },
+    {
+      "description": "BNB token on Carbon",
+      "denom_units": [
         {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/bneo.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/bneo.svg",
-          "theme": {
-            "primary_color_hex": "#040404"
-          }
+          "denom": "brdg/1768794901f8a19c2ec795a5402653cef6cbfe6b3ec6398d39fc37de963cb667",
+          "exponent": 0
+        },
+        {
+          "denom": "bnb",
+          "exponent": 18,
+          "aliases": [
+            "BNB"
+          ]
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "brdg/1768794901f8a19c2ec795a5402653cef6cbfe6b3ec6398d39fc37de963cb667",
+      "name": "BNB",
+      "display": "bnb",
+      "symbol": "BNB",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "binancesmartchain",
+            "base_denom": "wei"
+          },
+          "provider": "Axelar Bridge"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/binancesmartchain/images/bnb.svg"
+      },
+      "coingecko_id": "binancecoin"
+    },
+    {
+      "description": "cmETH token on Carbon",
+      "denom_units": [
+        {
+          "denom": "brdg/235ea1e9c215addc00464bfbb7f69c23100e38f3d5b8512b2a1e2fb1da1d7453",
+          "exponent": 0
+        },
+        {
+          "denom": "cmeth",
+          "exponent": 18,
+          "aliases": [
+            "cmETH"
+          ]
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "base": "brdg/235ea1e9c215addc00464bfbb7f69c23100e38f3d5b8512b2a1e2fb1da1d7453",
+      "name": "cmETH",
+      "display": "cmeth",
+      "symbol": "cmETH",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "mantle",
+            "base_denom": "0xE6829d9a7eE3040e1276Fa75293Bde931859e8fA"
+          },
+          "provider": "Axelar Bridge"
         }
       ],
-      "type_asset": "sdk.coin"
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/cmeth.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/carbon/images/cmeth.svg"
+      },
+      "coingecko_id": "mantle-restaked-eth"
```

Comment:
To save time, I'll mention that all bridged assets may not use the same coingecko id as the origin asset. Please remove most (if not all??) of the coingecko ids added. If the original asset was just IBC transferred over then it can stay.